tpp-run/MLIRBench.cpp

```diff
@@ -331,7 +359,8 @@ llvm::StringRef MLIRBench::createGlobal(MemRefType type) {
   auto tensorType =
       RankedTensorType::get(memrefTy.getShape(), memrefTy.getElementType());
   auto floatInit = mlir::DenseElementsAttr::get(tensorType, floatValue);
-  auto alignment = builder.getIntegerAttr(builder.getI64Type(), 128);
+  auto floatType = IntegerType::get(builder.getContext(), 64);
```

Comment:
Float type? This is the alignment as an integer.

Also, why the change from `getI64Type()` to `IntegerType::get(64)`? They yield exactly the same result.

Reply:
This is a stray change I forgot to revert locally. Will revert this, my bad, sorry.